src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -63,12 +63,20 @@ if (__DEV__) {
       null
     );
 
+    // Suggest an event name correction if possible
     warning(
       registrationName == null,
-      'Unknown event handler property %s. Did you mean `%s`?',
+      'Unable to register unsupported event handler property %s. Did you mean %s?',
       name,
       registrationName
     );
+
+    // Otherwise at least make it clear that the attributes will be filtered out
+    warning(
+      standardName != null || registrationName != null,
+      'Unable to assign unsupported DOM property %s.',
+      name
+    );
   };
 }
```

Comment:
The problem is that this error message doesn't specify the line number, as per the last sentence of my comment in https://github.com/facebook/react/pull/6459#issuecomment-207506815